src/thunderbird_accounts/mail/tasks.py

```diff
@@ -177,27 +206,58 @@ def create_stalwart_account(
         ],
     ]
 
+    _stalwart_check_or_create_domain_entry(stalwart, domain)
+    for alias in emails[1:]:
+        _domain = alias.split('@')[1]
+        print(f'checking domain {_domain}')
+        _stalwart_check_or_create_domain_entry(stalwart, _domain)
```

Comment:
Ran into an issue where alias domains weren't added. So this "fixes it" (It kinda sucks eh?)